pkg/cmd/enclave_secrets.go

```diff
@@ -95,11 +94,10 @@ var secretsSetCmd = &cobra.Command{
 	Long: `Set the value of one or more secrets.
 
 Ex: set the secrets "api_key" and "crypto_key":
-doppler secrets set api_key=123 crypto_key=456`,
+doppler enclave secrets set api_key=123 crypto_key=456`,
```

Comment:
can we use example secrets where the name is all caps?

Reply:
updated the examples